Codegen/test/resources/input/CobolV6/DynamicAllocationStatements.rdz.cbl

```diff
@@ -0,0 +1,22 @@
+﻿000000 IDENTIFICATION DIVISION.
+000000 PROGRAM-ID. FreeStatements.
+000000
+000000 DATA DIVISION.
+000000 WORKING-STORAGE SECTION.
+000000 01 var0 POINTER.
+000000 01 var1 POINTER.
+000000 01 var2 POINTER.
+000000 01 var3 POINTER.
+000000 01 var4.
+000000   05 var5 POINTER.
+000000 01 var6.
+000000   05 var7 POINTER.
+000000
+000000 PROCEDURE DIVISION.
+000000     FREE var0
+000000     FREE var1 var2 var3
+000000     FREE var5 OF var4
+000000     FREE var7 IN var6
```

Comment:
It will be nice to also use a TypeCobol feature like: FREE var4::var5 and FREE var6::var7

Reply:
See 985dc0d39396202ff2509ab5aa5e3abe930a0e55.